### PR TITLE
Add keyword input file argument

### PR DESCRIPTION
**Pull request by Timple**
**Description**
Compatibility with `urdf2webots` regarding the `--input` syntax.

**Related Issues**
-

**Affected Packages**
List of affected packages:
  - webots_ros2_importer

### CODE REVIEW

File: webots_ros2_importer/package.xml [hunk 1]
```diff
@@ -22,9 +22,6 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pycodestyle</test_depend>
-  <test_depend>python3-pil</test_depend>
-  <test_depend>python-numpy</test_depend>
-  <test_depend>python-imaging</test_depend>
```
Comment (lukicdarkoo) [inline webots_ros2_importer/package.xml:27]: Why deleting this?

Comment (Timple) [reply webots_ros2_importer/package.xml:27]: From the commit message it seems that those broke python3 compatibility?

@yanick-douven any insights?

**Review by BenjaminHug8 — APPROVED** [review]
It works fine, thank you !

**Comment by lukicdarkoo** [comment]
@Timple Feel free to merge it, you should have permissions

**Comment by Timple** [comment]
Thank you: https://github.com/ros/rosdistro/pull/31159

**Pull request merged**